viewer/reveal.api.md

```diff
@@ -784,15 +785,22 @@ export interface Image360 {
     readonly transform: THREE.Matrix4;
 }
 
+// @public
+export type Image360AnnotationHoveredDelegate = (annotation: AnnotationModel) => void;
```

Comment:
We should probably wrap the AnnotationModel from the SDK as I suspect we want to add more stuff such as styling eventually. Ok to do this in a separate PR though.

Reply:
Will do in a later PR :+1: https://cognitedata.atlassian.net/browse/REV-898